invisible_cities/reco/deconv_functions_test.py

```diff
@@ -248,10 +297,214 @@ def test_nonexact_binning(data_hdst, data_hdst_deconvolved):
     psf['zr']     = [z] * len(xx)
     psf           = pd.DataFrame(psf)
 
-    deco          = deconvolutor((h.X, h.Y), h.Q, psf)
+    deco          = deconvolutor((h.X, h.Y), h.Q, psf,
+                                 **no_satellite_killer)
 
     check_x = np.diff(np.sort(np.unique(deco[1][0]), axis=None))
     check_y = np.diff(np.sort(np.unique(deco[1][1]), axis=None))
 
     assert(np.all(check_x % 9 == 0))
     assert(np.all(check_y % 9 == 0))
+
+
+@mark.parametrize("cut_type", cut_types)
```

Comment:
```suggestion
@mark.parametrize("cut_type", CutType)
```
I think this should work. If so, you could remove the `cut_type = list(CutType)` line at the top of the file.